ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -47,17 +48,16 @@ class FaultInjector
     VelocityLimit=7, 
     NoForceData=8};
 
-  enum SystemFaults {
-    System=1, 
-    ArmGoalError=2, 
-    ArmExecutionError=4,
-    TaskGoalError=8, 
-    CamGoalError=16, 
-    CamExecutionError=32, 
-    PtGoalError=64, 
-    PtExecutionError=128, 
-    LanderExecutionError = 256};
-
+	static constexpr std::bitset<9> isSystem{		0b0'0000'0001 };
```

Comment:
What does System mean?

Reply:
System is just one of the fault types the JPL defined. I assume it means a general system fault?

Reply:
I figured that you were just copying the JPL values for this and the others I pointed out.  That's fine.  Would be good to find out what they mean, and how we can utilize them in our testbed.

Reply:
Yea I think some of these are future faults we need to build, like the camera fault. Some of them I'm not sure if we will need to target, like any goal errors. In which case we can just get rid of those